src/dune/__main__.py

```diff
@@ -279,8 +279,7 @@ def load_all_modules_from_dir(plugin_dir):
 # Integration with antler-proj begin ----------------------------------------------------------------------
 
             elif args.create_project:
-                dune_sys.create_project(args.create_project[1],
-                                        args.create_project[0], args.create_project[2])
+                dune_sys.create_project(*args.create_project)
```

Comment:
should we add here validation of `len(args.create_project)`?

Reply:
> should we add here validation of `len(args.create_project)`?

Added.